public/javascripts/components/nav.js

```diff
@@ -6,29 +6,111 @@ var navbar = Vue.extend({
           <h1 v-link="{path: '/'}" class="cc-logo">Chubbo-Chat</h1>
         </div>
         <div class="col-xs-4">
-          <p v-show="loginStatus" v-on:click='handleLogin' class="cc-loginBtn">login</p>
-          <p v-else v-on:click='handleLogout' class="cc-logoutBtn">logout</p>
+          <img
+            v-on:click='handleMenu'
+            src="/images/hamburger.svg"
+            class="cc-menuIcon"
+            id={{menuIconId}} 
+          />
+          <div id={{loginButtonsId}}>
+            <p
+              v-show="!user"
+              v-on:click='handleLogin'
+              class="cc-loginBtn"
+            >
+              login
+            </p>
+            <p
+              v-else
+              v-on:click='handleLogout'
+              class="cc-logoutBtn"
+            >
+              logout
+            </p>
+          </div>
+          <p v-link="{path: '/dashboard'}" id={{dashButtonId}}>
+            dashboard
+          </p>
         </div>
       </div>
     </div>
   `,
   data: function() {
-    return {loginStatus: firebase.auth().currentUser};
+    return {
+      user: firebase.auth().currentUser
+    };
+  },
+  ready: function() {
+    var me = this;
```

Comment:
I am glad you figured out this trick! Believe it or not, if you use an arrow () => function, you can access this.user directly from inside the setTimeout!